ui/raidboss/data/05-shb/raid/e9s.ts

```diff
@@ -173,18 +192,20 @@ const calculateSummonSafeZone = (boss, clone1, clone2, abilityId) => {
     .filter((pos) => !badZones.includes(pos))
     .map((pos) => directions[pos]);
 
-  if (safeZones.length !== 1)
+  const safeZone = safeZones[0];
+  if (safeZone === undefined || safeZones.length !== 1)
```

Comment:
```suggestion
  if (safeZones?.length !== 1)
```

Reply:
Good try on this one, but the `safeZone` is used below and so needs to be defined and have the right length.